src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -587,6 +589,10 @@
                                 {:on-press #(re-frame/dispatch [:chat.ui/save-image-to-gallery (:image content)])
                                  :id       :save
                                  :label    (i18n/label :t/save)}]
+                               (when platform/android?
```

Comment:
why only Android?

Reply:
on iOS there is this issue pointed out here https://github.com/status-im/status-react/pull/13113#issuecomment-1055398524

Reply:
why can't we fix it ?

Reply:
I looked into it. I didnt figure out  the bug. I thought I activate this for android first then have an issue for the ios

Reply:
i don't see if its a bug , it's more like wrong implementation